packages/nodes-base/nodes/Hubspot/CompanyDescription.ts

```diff
@@ -896,10 +896,7 @@ export const companyFields = [
 					loadOptionsMethod: 'getCompanyProperties',
 				},
 				default: '',
-				description: `Used to include specific company properties in the results.<br/>
-				By default, the results will only include company ID and will not include the values for any properties for your companys.<br/>
-				Including this parameter will include the data for the specified property in the results.<br/>
-				You can include this parameter multiple times to request multiple properties separed by ,.`,
+				description: `Used to include specific company properties in the results. By default, the results will only include company ID and will not include the values for any properties for your companys. Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by ,.`,
```

Comment:
```suggestion
				description: `<p>Used to include specific company properties in the results. By default, the results will only include company ID and will not include the values for any properties for your company.</p> <p>Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by <code>,</code>.</p>`,
```

Reply:
What was meant by `companys`? I assume they meant `company` not `companies` or `company's`

Reply:
Other params here have the same wording but mention `deals` and `tickets` so it's plural.